chipy_org/apps/meetings/views.py

```diff
@@ -39,6 +42,55 @@
 logger = logging.getLogger(__name__)
 
 
+class InitialRSVPMixin():
```

Comment:
This is a really interesting refactor. I'll admit that I'm not following all of what/why this class exists, but it makes sense that this is abstracting some of the logic around the "setup" rather than RSVP management.

Reply:
Yeah, this is one we should discuss.  Everything works, but I don't like that not super easy to follow.  

I abstracted the setup as I needed the logic for both the home page and the meeting detail page.  They instantiate the form, but don't use a form view.  

I had an earlier commit that provided a GET method and then used JavaScript on the fontend to insert the form into the DOM and add the CAPTCHA as well.  Which I thought was hard to follow...  :frown:

Regardless, it's an improvement.  I would do much more with it until I know the direction we want to push the site   For example a list of upcoming meetings and the ability to RSVPs to each meeting listed.